### DISCUSSION TITLE
Rock 5b+ ubuntu dont see bluetooth devices.

### DISCUSSION THREAD

**Discussion by tpnthr** [Joshua-Riek/ubuntu-rockchip#1018] [General]
Rock 5b+ ubuntu don't see Bluetooth devices.
Bluetooth device is discovered
Services are running 
Blueman is installed, but still no devices shown nearby.
Can anyone help me with that?

**Comment by tpnthr** [comment]
I think it is better to move this to issues section.

**Comment by seyko2** [comment]
ubuntu 22 (kernel 5.10), orange pi 3b (revision 2.1): can't switch bluetooth on. Why there are problems with bluetooth too?
